.travis.yml

```diff
@@ -33,29 +34,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.8.*
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=3.0.*
```

Comment:
php 7.0 is still in the list of php versions at the top. but i think sonata correctly does not run every permutation of configuration x every php version. rather have one build per php version + one build with the latest php for every configuration.